data/json/vitrification/unvitrified_farm.json

```
@@ -0,0 +1,288 @@
+        "    -.............-     ",
+        "    -.............-     ",
+        "    -.............-     ",
+        "    ---5..........-     ",
+        "       -..........-     ",
+        "       -..........-     ",
+        "       -----......-     ",
+        "           -......-     ",
+        "           --------     ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "palettes": [ "roof_palette" ],
+      "terrain": { ".": "t_shingle_flat_roof" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "unvitrified_orchard" ],
+    "weight": 100,
+    "object": {
+      "set": [ { "point": "variable", "id": "unvitrified_orchard_center", "x": 12, "y": 12 } ],
+      "fill_ter": "t_dirt",
+      "rows": [
+        "111111111111111111111111",
+        "!**********************1",
+        "!*    *    **    *    *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!**7     7****7     7**1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*    *    **    *    *1",
+        "@**********************1",
+        "@**********************1",
+        "!*    *    **    *    *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!**7     7****7     7**1",
+        "!*         **         *1",
+        "!*         **         *1",
+        "!* 7  7  7 ** 7  7  7 *1",
+        "!*    *    **    *    *1",
+        "!**********************1",
+        "111111111111111111111111"
+      ],
+      "palettes": [ "orchard_tree_apple" ],
+      "terrain": { "1": "t_quietfarm_border_fence", "!": "t_fence", "@": "t_fencegate_c" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates head of bed",
+    "nested_mapgen_id": "bedroom_4x4_child_vit",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "LEEL",
+        " EE ",
+        "y   ",
+        "O   "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "O": [ { "item": "SUS_dresser_mens", "chance": 50 }, { "item": "SUS_dresser_womens", "chance": 50, "repeat": [ 1, 2 ] } ],
+        "E": { "item": "bed", "chance": 40, "repeat": [ 1, 2 ] },
+        "I": { "item": "SUS_desks_bedroom_unisex", "chance": 40, "repeat": [ 1, 2 ] },
+        "L": { "item": "bedroom", "chance": 20, "repeat": [ 1, 2 ] }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates head of bed",
+    "nested_mapgen_id": "bedroom_5x4_adult_unvitrified",
```

```suggestion
    "method": "json",
    "nested_mapgen_id": "bedroom_5x4_adult_unvitrified",
```
This is named 5x4 incorrectly